cc/modules/protocol/public/include/protocol_ops.h

```diff
@@ -371,36 +392,13 @@ class ProtocolOps {
     THROW_NOT_IMPL;
   }
 
-  // get \sqrt{a}
-  virtual int Sqrt(
-    const vector<string>& a,
-    vector<string>& output,
-    const attr_type* attr_info = nullptr) {
-    THROW_NOT_IMPL;
-  }
-
-  // get 1/\sqrt{a}
```

Comment:
why these comments are removed?

Reply:
I will  add them again and described accurately.